lib/reducers/onboarding.js

```diff
@@ -13,8 +13,9 @@ function onboardingReducer (state = initialState, action) {
       return { ...state, userData }
     case WIPE_ONBOARDING_DATA:
       return initialState
-    case SET_VERIFICATION_STATE:
-      return Object.assign({}, state, {verificationState: action.verificationState})
+    case SET_VERIFICATION_STATE:            
+      state.verificationState = action.verificationState
```

Comment:
In redux you use the immutable pattern which is why we use Object.assign which takes an empty object as the first param, assigns all the keys in state to it, and then overwrites verificationState. This shouldn't be changed